src/App.jsx

```diff
@@ -14,70 +17,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [currentGood, setCurrentGood] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
Not critical, but would be great to follow the instruction
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/2232b1ef-8a2f-4cf3-83a1-68ac87a1d89f)